LICENSE

```diff
@@ -1,4 +1,11 @@
-Copyright (c) 2020 Symmetry Investments, with portion (c) 2012-2016 Nicolas Gurrola
+Parts of the code (identified by comments within) fall within these other licenses:
+
+Copyright (c) 2014-2022 MongoDB specification maintainers
```

Comment:
it is the crud.d that does this right? this license seems to prohibit symmetry from using it.....